lib/OpenQA/WebAPI/Controller/API/V1/Table.pm

```diff
@@ -163,13 +163,20 @@ sub create {
 
     for my $par (@{$tables{$table}->{required}}) {
         $validation->required($par);
-        $entry{$par} = $self->param($par);
+        if (!defined $self->param($par)) {
+            next;
+        }
+        my $value = $self->param($par);
+        $value =~ s/^\s+|\s+$//g;
+        $entry{$par} = $value;
```

Comment:
I think it'd be a good idea to move getting the entries, settings and validation here to a separate method shared with the `update` sub because they do the exact same thing.

Reply:
Maybe I should trim white space in this PR, after all according to Coolo's comments, I need to trim  white space in openqa.ini and workers.ini. Then I could create a new PR to improve the code about the repeated operation. Do you think it is ok?

Reply:
I'm just suggesting to move it in a sub, since you're making the exact same fix twice.

Although as long as you're adding the appropriate unit tests I don't mind if you do it in two PRs, your call.

Reply:
I fix it in this PR. I think maybe it is convenient for merging code.